### PR TITLE
Database users

### PR DESCRIPTION
**Pull request by jonycfu**
Merge request. Users.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+class UsersController < ApplicationController
+
+  def update
+    @user = User.find(params[:id])
+
+    if @user.nil?
+      render json: {message: "User Not Found"}, status: :not_found
+    else
+      @user.update(user_params)
```
Comment (f3r) [inline app/controllers/users_controller.rb]:
``` ruby
if @user.update(user_params)
  render :show
else
  render json: {message: @user.error.messages}, status: :bad_request
end
```

https://speakerdeck.com/fermartin/ga-rails-activerecord

File: app/views/users/show.jbuilder [hunk 2]
```diff
@@ -0,0 +1,12 @@
+json.id @user.id
```
Comment (f3r) [inline app/views/users/show.jbuilder]: indentation would be awesome

**Comment by f3r** [comment]
fix those issues and merge when ready :) thanks!

**Comment by jonycfu** [comment]
@f3r Issues fixed. Ready for merge.

**Comment by jonycfu** [comment]
@f3r branch ready for merge

**Pull request merged**